assets/css/main.css

```diff
@@ -0,0 +1,996 @@
+body {
+  font-family: Arial, Helvetica, "Montserrat", sans-serif !important;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  font-family: "Roboto Condensed", sans-serif !important;
+}
+
+.nav-header .header-top {
+  background-color: #f72210 !important;
+  height: 95px;
+  padding: 20px 10px 15px;
```

Comment:
[REQUIRED] Your CSS file looks good but you are using the `!important` 29 times. We should avoid using !important as much as we could. It is not a good practice to over use it. Let's remove it from the styling.

Reply:
I am using !important because of bootstrap I can't overide bootstrap class without using it